Dshop/apps/users/tests/test_api_password_change.py

```diff
@@ -0,0 +1,124 @@
+import json
+
+import pytest
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+
+User = get_user_model()
+
+
+@pytest.fixture
+def password_change_url():
+    return reverse('api-password-change')
+
+
+@pytest.fixture
+def login_data():
+    return {
+        'username': 'testuser',
+        'password': 'testpassword',
+    }
+
+
+@pytest.fixture
+def password_change_data():
+    return {
+        'current_password': 'testpassword',
+        'new_password': 'new_testpassword',
+        'new_password_again': 'new_testpassword',
+    }
+
+
+@pytest.fixture
+def user_instance(login_data):
+    return User.objects.create_user(**login_data)
+
+
+@pytest.fixture
+def user_instance_token(user_instance):
+    return Token.objects.get_or_create(user=user_instance)[0]
+
+
+@pytest.mark.django_db
+def test_password_change_success(api_client, password_change_url, user_instance_token, password_change_data):

```

Comment:
Please make it two separate tests
1. success
- Would be good to show that I can log in with new password. 
3.  failure.